package/bluez5_utils: bump to version 5.83 and fix LE connection abort

bdraco opened the pull request:
- Bump BlueZ from 5.79 to 5.83
- Add patch to fix device removal on LE connection abort (from upstream PR #1521) This fixes an issue where temporary devices are incorrectly removed from D-Bus after connection failures, preventing retry attempts.

Fixes: https://github.com/bluez/bluez/issues/1489

sairon commented on package/bluez5_utils/0005-device-Fix-device-removal-on-le-connection-abort.patch:
```diff
@@ -0,0 +1,59 @@
++		}
+ 
+ 		if (device_get_auto_connect(device)) {
+ 			DBG("Enabling automatic connections");
```
The patch has bad formatting, the missing newline prevents it from applying. With that fixed, it still applies with an offset on clean 5.83 sources:
```
Applying 0005-device-Fix-device-removal-on-le-connection-abort.patch using patch: 
patching file src/device.c
Hunk #1 succeeded at 6233 (offset -38 lines)
```

sairon commented on package/bluez5_utils/0005-device-Fix-device-removal-on-le-connection-abort.patch:
```diff
@@ -0,0 +1,59 @@
+With this fix, the device remains available for the retry, allowing
+the connection to succeed on subsequent attempts.
+
+Fixes: https://github.com/bluez/bluez/issues/1489
```
A nit, but the patch also should have `Upstream:` annotation, pointing to the ML post: https://buildroot.org/downloads/manual/manual.html#_additional_patch_documentation

sairon reviewed (changes requested):
The existing patches don't apply anymore - the first two have been merged to 5.80, the rest to 5.83 - so they can be all deleted.

There are some (easily fixable) issues with the patch itself, as pointed out below.

Also, it would be better to split this into two commits - one doing the bump and removing the patches, the other adding the patch. Last nit is that the commits to Buildroot itself should have `Signed-off-by` line too - we don't enforce it but if you wanted to upstream the patches, it's mandatory there. Ideally just keep this PR opened and force-push the new two commits, so the PR can be FF-merged to our branch.

sairon committed:
package/{bluez5_utils, bluez5_utils-headers}: bump to version 5.83

Removed patches merged upstream in 5.80 (first two) and 5.83.

Changelog:
https://git.kernel.org/pub/scm/bluetooth/bluez.git/tree/ChangeLog

Co-authored-by: J. Nick Koston <nick@koston.org>
Signed-off-by: Jan Čermák <sairon@sairon.cz>

sairon committed:
package/bluez5_utils: add patch fixing device removal on le-connection-abort-by-local

Add yet unmerged patch for incorrect device removal after
le-connection-abort-by-local event, which can result in inability to retry
connection under some circumstances.

Fixes: https://github.com/bluez/bluez/issues/1489
Co-authored-by: J. Nick Koston <nick@koston.org>
Signed-off-by: Jan Čermák <sairon@sairon.cz>

sairon reviewed (approved):
Split into two commits and updated the patches. Adding @agners for another look.

agners reviewed (approved):
LGTM!
